client.js

```diff
@@ -12,16 +12,16 @@ imFeelingLucky.addEventListener("submit", imFeelingLuckyFunc);
 
 function googleSearchFunc(e) {
     e.preventDefault();
-    const title = document.getElementById("title");
-    const image = document.getElementById("image");
-    const text = document.getElementById("description");
+    const title = document.getElementById("title"); //
+    const image = document.getElementById("image"); //
+    const text = document.getElementById("description"); //
```

Comment:
These variables could be declared outside of the function to avoid repetition, the same constant variables are defined twice in this code